VEHICLE/SetConvertibleRoof.md

```diff
@@ -7,9 +7,39 @@ ns: VEHICLE
 // 0xF39C4F538B5124C2 0xC87B6A51
 void SET_CONVERTIBLE_ROOF(Vehicle vehicle, BOOL p1);
 ```
-
+# This allows for the vehicle's roof to be put on when set to true, and removed when set to false, provided that the vehicle has a version with a roof and a version without a roof.
 
 ## Parameters
-* **vehicle**: 
-* **p1**: 
+* **vehicle**: The vehicle to which the roof state will be applied.
+* **p1**: Boolean value where `true` adds the roof and `false` removes it, applicable only if the vehicle has a version with and without a roof.
```

Comment:
Maybe rename `p1` to `toggle`, or `roofState`, but I believe this toggles a roof on/off.